ui/hooks/useGetAssetImageUrl.ts

```diff
@@ -0,0 +1,19 @@
+import { useState, useEffect } from 'react';
+import { getAssetImageURL } from '../helpers/utils/util';
+
+const useGetAssetImageUrl = (image: string | null, ipfsGateway: string) => {
```

Comment:
What's the thinking behind introducing this hook?  Is it downstream of `getFormattedIpfsUrl` becoming async?

Reply:
yes, created this hook to refactor/remove some of the repeated logic in some of the useEffects i have introduced